external-crates/move/crates/move-package/src/resolution/resolving_table.rs

```diff
@@ -30,6 +30,12 @@ enum Assignment {
     Linked(usize),
 }
 
+impl Default for ResolvingTable {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
Is this that stupid `new` without default warning? 
Could you remove this please :)